### PR TITLE
Update sass

### PR DESCRIPTION
**Pull request by bguiz**
- Previously v0.9.4
- Now v2.0.1
- Hardly any breaking changes: Only minor ones required in success and error callbacks

### CODE REVIEW

**Comment by impaler** [comment]
Thanks for this. I think we need to add some more coverage to the work Ben did on the sass @imports in the integration tests before we merge.

Its a shame this version doesn't fix fix build errors for other versions of node, we only have version 0.10 for now https://travis-ci.org/angularity/node-angularity/jobs/51189499.

File: lib/build/node-sass.js [hunk 1]
```diff
@@ -139,10 +139,18 @@ module.exports = function (bannerWidth, libraryPaths) {
...
      * Handler for successful transpilation using node-sass.
-     * @param {string} css Compiled css
+     * This functions gets called with an object containing a CSS file and its sourcemap,
+     * which is modified and passed through CSS rework, before being pushed to the results.
+     *
+     * @param {object} results Results object from node-sass, which contains:
+     *   - css {string}
+     *   - map {string}
+     *   - stats {object}
      * @param {string} map The source-map for the compiled css
```
Comment (bholloway) [inline lib/build/node-sass.js]: You are looking for **record type**.
Per the [specification](http://usejsdoc.org/tags-type.html) I believe this `@param` would be written as follows:

 `@param {{css:string, map:string, stats:object}} results Results object from node-sass`

**Comment by bholloway** [comment]
@impaler I will take responsibility for manual testing these issues until you can get the integration tests sorted

**Comment by bholloway** [comment]
currently libsass (and therefore node-sass) has broken source maps, we will revisit when the situation changes

**Pull request closed without merging**